src/Parser.js

```diff
@@ -47,7 +47,7 @@ var ParseError = require("./ParseError");
  */
 function Parser(input, settings) {
     // Make a new lexer
-    this.lexer = new Lexer(input);
+    this.gullet = new MacroExpander(input, settings && settings.macros);
```

Comment:
It's probably worth putting a comment here too about why it's called the `gullet`.